src/pani/async.cljs

```diff
@@ -0,0 +1,85 @@
+(ns pani.async
+  (:require [pani.core :as p]
+            [cljs.core.async :refer [<! >! chan put! close!]])
+  (:require-macros [cljs.core.async.macros :refer [go go-loop]]))
+
+;; (defn chan-val [] (chan (map p/value)))
+
+;; (defn chan-snap [] (chan (map p/wrap-snapshot)))
+
+(defn with-chan "Dumb pseudo-macro" [f]
```

Comment:
Style guide shaming! I'm more embarrassed by the function than the styling here though :)